integration/spark-common/src/main/scala/org/apache/spark/sql/hive/DistributionUtil.scala

```diff
@@ -115,7 +115,9 @@ object DistributionUtil {
       sparkContext: SparkContext): Seq[String] = {
     val confExecutors: Int = getConfiguredExecutors(sparkContext)
     LOGGER.info(s"Executors configured : $confExecutors")
-    val requiredExecutors = if (nodesOfData < 1 || nodesOfData > confExecutors) {
+    val requiredExecutors = if (nodesOfData < 1) {
+      1
+    } else if (nodesOfData > confExecutors) {
```

Comment:
this change duplicates [PR-1241](https://github.com/apache/carbondata/pull/1241)

Reply:
This is worked over 1241.When 1241 is merged, this change will not reflect